src/factories/details/ProductionView.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  FactoryInputIcon,
+  FactoryOutputIcon,
+} from '@/factories/components/peek/icons/OutputInputIcons';
+import { FactoryOutputRow } from '@/factories/inputs/output-row/FactoryOutputRow';
+
+const progressValues: { value: FactoryProgressStatus; label: string }[] = [
+  {
+    value: 'draft',
+    label: 'Draft',
+  },
+  {
+    value: 'todo',
+    label: 'Todo',
+  },
+  {
+    value: 'in_progress',
+    label: 'In progress',
+  },
+  {
+    value: 'done',
+    label: 'Done',
+  },
+];
+
+export const ProductionView = ({ id }: { id: string }) => {
+  const factory = useFactorySimpleAttributes(id);
+  const { inputs, outputs } = useFactoryInputsOutputs(id);
+  const update = useCallback(
+    (path: Path<Factory>, value: string | null | number) => {
+      useStore
+        .getState()
+        .updateFactory(id, state => setByPath(state, path, value));
+    },
+    [id],
+  );
+  const onChangeHandler = useFormOnChange<Factory>(update);
+  const status = factory.progress && progressProperties[factory.progress];
+
+  return (
+    <Container size="lg">
+      <Group gap="xl" align="start" py="xl">
+        <Stack gap="lg" style={{ flexGrow: 1 }}>
+          <Stack gap="sm">
+            <Text size="lg">Inputs</Text>
+            <Stack gap="xs">
+              {inputs?.map((input, i) => (
+                <FactoryInputRow
+                  key={i}
+                  index={i}
+                  input={input}
+                  factoryId={id!}
+                  onChangeHandler={onChangeHandler}
+                  displayMode="factory"
+                />
+              ))}
+            </Stack>
+
+            <Button
+              size="sm"
+              leftSection={<FactoryInputIcon />}
+              color="blue"
+              variant="light"
+              onClick={() => {
+                useStore.getState().addFactoryInput(id!);
+              }}
+            >
+              Add Input
+            </Button>
+          </Stack>
+          <Stack gap="sm">
+            <Text size="lg">Outputs</Text>
+
+            <Stack gap="xs">
+              {outputs?.map((output, i) => (
+                <FactoryOutputRow
+                  key={i}
+                  index={i}
+                  output={output}
+                  factoryId={id!}
+                  displayMode="factory"
+                />
+              ))}
+            </Stack>
+
+            <Button
+              size="sm"
+              leftSection={<FactoryOutputIcon />}
+              color="blue"
+              variant="filled"
+              onClick={() => {
+                useStore.getState().addFactoryOutput(id!);
+              }}
+            >
+              Add Output
+            </Button>
+          </Stack>
+        </Stack>
+        <Stack gap="sm" align="stretch" bg="dark" p="md">
+          <Text size="lg">Properties</Text>
+          <TextInput
+            value={factory?.name ?? 'Solver'}
```

Comment:
We could fall back to undefined (or `''` i don't remember if it allows undefined) in order to avoid showing "Solver"